route.go

```diff
@@ -47,6 +43,7 @@ type RouteLeg struct {
 type Annotation struct {
 	Duration []float32 `json:"duration"`
 	Distance []float32 `json:"distance"`
+	Nodes    []int64   `json:"nodes"`
```

Comment:
@Zverik Maybe use unsigned int64? WDYT?

Reply:
Does not matter bit-wise, since node ids are inside 2³³ now. But it might help fail on incorrect data. Negative ids is OSM are usually employed for newly created objects not yet uploaded to the main OSM database. I doubt OSRM works with these.

Reply:
@Zverik thx for the explanation.
@regeda made it unsigned.